port/cpl_conv.cpp

```diff
@@ -1808,6 +1795,29 @@ const char *CPL_STDCALL CPLGetThreadLocalConfigOption(const char *pszKey,
     return pszResult;
 }
 
+/************************************************************************/
+/*                   CPLGetGlobalConfigOption()                    */
+/************************************************************************/
+/** Same as CPLGetConfigOption() but excludes options set with
+ *  CPLSetThreadLocalConfigOption() */
```

Comment:
```suggestion
 *  CPLSetThreadLocalConfigOption().
 * This function should generally not be used by applications, which should rather
 * use CPLGetConfigOption()
 * @since 3.8
```

Reply:
Added. Also noted that it excludes environment variables.